src/login/logind-dbus.c

```diff
@@ -1488,59 +1495,32 @@ static int have_multiple_sessions(
         return false;
 }
 
-_printf_(2, 0)
-static int log_with_wall_message(Manager *m, const char *d, const char *p, const char *q) {
+static int bus_manager_log_shutdown(
+                Manager *m,
+                const ActionTableItem *a) {
+
+        const char *message = a->message;
+        const char *log_str = a->log_str;
+
         assert(m);
 
-        if (isempty(m->wall_message))
-                p = strjoina(p, ".");
+        if (message)
```

Comment:
`isempty()`?

Reply:
hmm, it's a static built in table, so unless someone intentionally puts "" in the source it's either NULL or something IMO.

Reply:
Ah, I see.